### DISCUSSION TITLE
KEDA is unable to connect to a Kafka broker running in my local k8s cluster

### DISCUSSION THREAD

**Discussion by khaledhikmat** [kedacore/keda#1530] [Q&A / Need Help]
Is there a known problem with Docker for Windows K8s cluster? I was able to get KEDA to work if I connect it to a Kafka broker running outside the local cluster. Of course, I am able to connect other consumers to my local broker without issues and I am using the same broker-service with KEDA `ScaledObject`. 

Thank you. This is a very interesting product.

**Comment by khaledhikmat** [comment]
Sorry ...I forgot to mention that the error I get is: `client has run out of available brokers to talk to (Is your cluster reachable?)` when I look into the KEDA operator logs.

**Reply by zroubalik** [reply]
Sorry, don't have any experience with Docker for Windows K8s cluster. Maybe @tomkerkhove ?

**Reply by tomkerkhove** [reply]
Unfortunately not, but that error typically means that it cannot reach the broker. Did you verify that?

**Reply by khaledhikmat** [reply]
Ok...thank you both.
